src/MohammadYounes.Owin.Security.MixedAuth/MixedAuthHandler.cs

```diff
@@ -58,9 +59,31 @@ public MixedAuthHandler()
                     // Microsoft.Owin.Security.AuthenticationManagerExtensions: ExternalLoginInfo GetExternalLoginInfo(AuthenticateResult result)
                     claimsIdentity.AddClaim(new Claim(ClaimTypes.NameIdentifier, logonUserIdentity.User.Value, null, Options.AuthenticationType));
 
-                    //could grab email from AD and add it to the claims list.
+                    // do we need this ?
```

Comment:
Yes, this will be used as the provider key. Without it you can't identify which local account is linked to the windows identity.